packages/adapter-static/index.js

```diff
@@ -7,6 +7,13 @@ export default function (options) {
 		name: '@sveltejs/adapter-static',
 
 		async adapt(builder) {
+			if (builder.config.kit.router?.resolution === 'server' && options?.fallback) {
+				throw new Error(
+					'adapter-static with a fallback page cannot be used with `router.resolution: "server"`, ' +
+						'as this combination of configs requires a server to handle dynamic route resolution requests'
+				);
+			}
+
```

Comment:
hopefully this is no longer true if route modules are prerendered?

```suggestion
```

Reply:
No, because the fallback will only render a shell, so we can't crawl anything going from there, and as such we will likely not reach all pages, meaning server routes will be missing

Reply:
I don't understand, this is distinct from crawling surely?